core/arch/instr.h

```diff
@@ -1759,6 +1759,13 @@ DR_API
  */
 bool
 instr_zeroes_ymmh(instr_t *instr);
+
+DR_API
+/** Returns true if \p instr's opcode is OP_xsave32, OP_xsaveopt32, OP_xsave64,
+ * OP_xsaveopt64, OP_xsavec32 or OP_xsavec64.
```

Comment:
Use `#` to make links.  This also ensures you have no typo here b/c doxygen will complain about an invalid link in that case.

Reply:
Done.